sycl/include/sycl/ext/oneapi/device_global/device_global.hpp

```diff
@@ -58,11 +58,19 @@ class device_global_base {
   // The pointer member is mutable to avoid the compiler optimizing it out when
   // accessing const-qualified device_global variables.
   mutable pointer_t usmptr{};
+  T val {};
```

Comment:
Nit: Could we come up with a better name to avoid someone thinking this actually holds the value of it? Also, can we make it `const`?

Reply:
Would something like `InitValForUSMPtr` be a better name?

Reply:
It irks me that I named the member `usmptr` but I think we should at least follow the naming scheme for the new variable still, but something like `initval` or `init_val` would be enough for me. 😄